Sources/App/GithubService+IssueComment.swift

```diff
@@ -0,0 +1,89 @@
+import Vapor
+import Stevenson
+
+struct PingAction: Content {
+    let zen: String
+}
+
+struct CommentAction: Content {
+    let action: String
+    let comment: Comment
+    let issue: Issue
+
+    struct Comment: Content {
+        let body: String
+    }
+
+    struct Issue: Content {
+        let number: Int
+    }
+
+    struct PullRequest: Content {
+        let url: URL
+    }
+}
+
+extension GitHubService {
+    func webhook<T: Decodable>(from request: Request) throws -> Future<T> {
```

Comment:
Does this really needs to be generic, while the implementation seems to be very specific towards issue-comment webhooks, and likely only be used to always decode `CommentAction`?